src/lem/eval.rs

```diff
@@ -1319,4 +1796,52 @@ mod tests {
         store.hydrate_z_cache();
         test_eval_and_constrain_aux(step_fn, &store, pairs);
     }
+
+    #[test]
+    fn test_dumb_lang() {
+        use crate::coprocessor::test::DumbCoprocessor;
+
+        let mut lang = Lang::<Fr, DumbCoprocessor<Fr>>::new();
+        let dumb = DumbCoprocessor::new();
+        let name = user_sym("cproc-dumb");
+
+        let store = &mut Store::default();
+        lang.add_coprocessor_lem(name, dumb, store);
+        let func_ivc = make_eval_step_from_lang(&lang, true);
+        let func_nivc = make_eval_step_from_lang(&lang, false);
+        for func in [func_ivc, func_nivc] {
```

Comment:
I'd love to see some negative testing for an non-existing coproc as well.

Reply:
A non-existing coprocessor would just raise an error for an unbound variable. It's unrelated to the changes made in this PR

Reply:
I expected that result, I'm just saying we've spent quite a bit of time being very specific about the nature of the errors returned (not raised) by the evaluation, and we will want to document it as part of a unit test.

Reply:
I've added it, but I should say that these tests in `eval.rs` are meant to be temporary and will go away once we integrate the evaluation test suite

Reply:
thanks!